src/getTreeColumnWidth.ts

```diff
@@ -0,0 +1,79 @@
+import {
+  Cube,
+  DataVisualizationWidgetMapping,
+  LevelCoordinates,
+  getHierarchy,
+  getLevel,
+} from "@activeviam/activeui-sdk";
+import { quote } from "@activeviam/mdx";
+
+/**
+ * Returns the column width for the tree table calculated by the baseWidth + (maxLevelDepth * levelMultiplier)
+ */
+export function getTreeColumnWidth({
```

Comment:
It would be nice to rename this function into `getTreeColumnKey`, make it only responsible for returning the key and not the width and inline the computation of the width where it is needed, since it's a one liner and all the rest of this function here is concerned about the key of the tree column, not its width.

Reply:
Done 👍